app/scripts/configs/tracks-info.js

```diff
@@ -1509,17 +1571,24 @@ export const TRACKS_INFO = [
       'color',
       'stroke',
       'fontSize',
-      'fontIsAligned',
+      'fontIsLeftAligned',
```

Comment:
should this be `fontAlignment` rather than `fontIsLeftAligned`?

Reply:
Now I am thinking it should really be `textAlign` (to adhere to CSS) but right now it's `fontIsLeftAligned`. This change is in general unrelated to the PR but I noticed this typo some time ago and just fixed it. Oh and `fontIsAligned` does not exist so it didn't do anything before.